core/src/Cuda/Kokkos_Cuda_Instance.hpp

```diff
@@ -193,6 +194,338 @@ class CudaInternal {
     }
   }
 
+  // Using cudaAPI function/objects will be w.r.t. device 0 unless
+  // cudaSetDevice(device_id) is called with the correct device_id.
+  // The correct device_id is stored in the static variable
+  // CudaInternal::m_cudaDev set in Cuda::impl_initialize(). It is not
```

Comment:
Soon that variable will not be `static` anymore.

Reply:
I'll remove mention of static in the comment.